src/Core/Geographic/Coordinates.js

```diff
@@ -0,0 +1,170 @@
+/**
+ * Generated On: 2015-10-5
+ * Class: Coordinates
+ * Description: Coordonnées cartographiques
+ */
+
+import * as THREE from 'three';
+import mE from '../Math/MathExtended';
+import Ellipsoid from '../Math/Ellipsoid';
+
+export function ellipsoidSizes() {
+    return {
+        x: 6378137,
+        y: 6356752.3142451793,
+        z: 6378137,
+    };
+}
+
+const ellipsoid = new Ellipsoid(ellipsoidSizes());
+
+export const UNIT = {
+    RADIAN: 0,
+    DEGREE: 1,
+    METER: 2,
+};
+
+export const supportedCRS = {
+    'EPSG:4326': UNIT.DEGREE,
+    'EPSG:4326R': UNIT.RADIAN,
```

Comment:
`EPSG: 4326R` does not exist, as you said, try to avoid this notation.

maybe add `unit` properties : 

```js
function Coordinates(crs, ...coordinates) {
    assertCrsIsValid(crs);
    this.crs = crs;
    // Set unit by default unit 
    this.unit = supportedCRS[this.crs];
    this._values = new Float64Array(3);
    for (let i = 0; i < coordinates.length; i++) {
        this._values[i] = coordinates[i];
 }
}
Coordinates.prototype.unit = function unit() {
    return this.unit;
};
Coordinates.prototype.asUnit = function asUnit(unit) {
    const cC = this.clone();
    cC.unit = unit;
    return cC;
};
Coordinates.prototype.clone = function clone() {
    return new Coordinates(this.crs, this._values[0], this._values[1], this._values[2]);
};
```

and use like this 

```js
const coordinates = new Coordinates('EPSG:4326', 10, 40, 0).asUnit(UNIT.RADIAN);
```